ros-conversion/v2x-ros-conversion/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<!--
+  Copyright (C) 2018-2024 LEIDOS.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<package format="3">
+  <name>v2x-ros-conversion</name>
+  <version>1.0.0</version>
+  <description>Meta-package for launching a full v2x-ros-conversion instance</description>
+
+  <maintainer email="carma@dot.gov">carma</maintainer>
+
+  <license>Apache 2.0</license>
+
+  <!-- TODO once all of carma-messenger is converted to ROS2 add actual exec dependencies to package.xml -->
```

Comment:
Let's remove this comment and add the exec dependencies like here https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/carma_wm_ctrl/package.xml